pkgs/servers/home-assistant/custom-components/climate_group/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+}:
+buildHomeAssistantComponent rec {
+  owner = "bjrnptrsn";
+  domain = "climate_group";
+  version = "1.0.7";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = domain;
+    rev = "refs/tags/${version}";
+    hash = "sha256-f/VQUNzRSxmKGNgijaafQ5NbngUUKmcdkafYC3Ol9qM=";
+  };
+
+  dontBuild = true;
+
+  meta = with lib; {
+    changelog = "https://github.com/bjrnptrsn/climate_group/releases/tag/${version}";
```

Comment:
Why not
```suggestion
    changelog = "https://github.com/bjrnptrsn/climate_group/blob/${src.rev}/README.md#changelog";
```
?

Reply:
They have the same content, but the releases seemed more standard
- https://github.com/bjrnptrsn/climate_group/releases/tag/1.0.7
- https://github.com/bjrnptrsn/climate_group#107

Reply:
Most projects don't have an actual changelog but I'd say if they do it's preferred because it allows for an easier overview of the changes if we skip a release.